src/main/java/org/apache/sling/discovery/base/commons/DefaultTopologyView.java

```diff
@@ -119,7 +119,7 @@ public Type compareTopology(final DefaultTopologyView other) {
 
     @Override
     public boolean equals(final Object obj) {
-        if (obj == null || !(obj instanceof DefaultTopologyView)) {
+        if (!(obj instanceof DefaultTopologyView)) {
```

Comment:
I see why this is identical to the previous code, but maybe it's just me, for me the original code reads easier. I would prefer the original here.

Reply:
thx! (ah, sonar! :)